src/StorageApiHandler.php

```diff
@@ -7,20 +7,17 @@
 use Keboola\DockerBundle\Monolog\Handler\StorageApiHandlerInterface;
 use Keboola\StorageApi\Client;
 use Keboola\StorageApi\Event;
-use function Keboola\Utils\sanitizeUtf8;
 use Monolog\Handler\AbstractHandler;
 use Monolog\Logger;
+use function Keboola\Utils\sanitizeUtf8;
 
 class StorageApiHandler extends AbstractHandler implements StorageApiHandlerInterface
 {
-    /** @var string */
-    protected $appName;
-
-    /** @var Client */
-    protected $storageApiClient;
+    protected string $appName;
+    protected Client $storageApiClient;
 
-    /** @var array */
-    private $verbosity;
+    /** @var array<int|string, string> */
```

Comment:
Stejny typ by idelane mel byt i u getteru/setteru. Jinak podle dokumentace PHP:
> The key can either be an int or a string. The value can be of any type.

Tim padem `array<string>` by melo byt enough, protoze technicky do klice stejne nic jinyho nez `int|string` nedostanes.

Reply:
jasně, mě tam prudilo phpcs s array typehintama a to je zapomenutý